articles/performance-optimization-techniques-for-node-js/index.md

```diff
@@ -307,14 +308,16 @@ You may consider using [Google Analytics](https://analytics.google.com/analytics
 Node.js monitoring frameworks include [Hapi](https://github.com/hapijs/hapi), [Restify](http://restify.com/), [Derby](https://derbyjs.com/), [Express](https://expressjs.com/), [Koa](https://koajs.com/), and [Mocha](https://mochajs.org/).
 
 ### Keeping your Code Light and Compact
-When developing mobile apps, make sure you apply the concept of making your codebase small and compact for both the client code and server code. This will reduce latency and increase speed for load times.
+When developing mobile and web apps, make sure you apply the concept of making your codebase small and compact for both the client code and server code. This will reduce latency and increase speed for load time.
```

Comment:
How about instead of "increase speed for load time", you could say, "reduce load times"